### ISSUE TITLE
The language is identified in the chat widget

### ISSUE THREAD

**Issue by kerlikur** [buerokratt/Machine-Translation-to-Buerokratt#2]
AS A Bürokratt
I WANT TO have the language of the message identified in the chat widget
SO THAT I can make further decisions

### Acceptance Criteria:
- [ ] The language is identified (https://pypi.org/project/langdetect/, fastText, langid.py) before taking further steps.
- [ ] If the machine translation of the language is available, continue the conversation in the detected language. If not, allow the end user to choose another language. #6
- [ ]  Bürokratt is able to detect the language of the message.

**Comment by turnerrainer** [comment]
@kerlikur should the language be detected
1. only once and then saved for later use from this point on;
2. every time the user provides new input?

**Comment by turnerrainer** [comment]
@kerlikur should this feature take effect

1. only after the input has been passed by the user (someone pressed enter); or
2. while (potentially) showing CSA in real time what the client is currently typing?

We don't have the second functionality yet but is in our backlog. But I strongly suggest the first selection here.

**Comment by turnerrainer** [comment]
@kerlikur 
>If the machine translation of the language exists, continue the conversation. If not, let the end user choose another language.

How would that work? If someone types in Finnish, which we don't support, should we recommend the client type the same thing in English? That's probably not meant like this but needs some clarification as I'm a bit confused.

**Comment by turnerrainer** [comment]
@kerlikur 
>The chat widget (Bürokratt) understands the language of the message.

1. Remove "The chat widget", just "Bürokratt understands the language of the message" would be correct;
2. This needs a separate issue as we need to implement a language detection library as well.

**Comment by kerlikur** [comment]
@turnerrainer 
The language must be identified after each message sent not at the same time when use writes. Only when the message is sent, no real-time typing needed.
If the machine translation of the language exists, continue the conversation. If not, let the end user select the appropriate one from the available machine translation languages. Currently there are 6 language directions to choose from #5 .
